.github/workflows/elixir-ci-v2.yml

```diff
@@ -0,0 +1,85 @@
+name: Elixir Quality CI
+
+
+on:
+  workflow_call:
+    inputs:
+          # -- Elixir version -- #
+          ci-container-version:
+            description: Which version of Elixir to use for the CI
+            default: elixir:1.8-otp-22-alpine
+            required: false
+            type: string
+          # -- Ignore steps -- #
+          ignore-credo:
+            description: ignore lint
+            default: true
+            required: false
+            type: boolean
+          # -- Services -- #
+          redis-version:
+            description: Redis version to use in tests. Ignored if empty
+            default: ""
+            required: false
+            type: string
+            
+permissions:
+  contents: read
+  
+jobs:
+  build:
+    name: Formatting, Credo, Sobelow, Deps audit, Tests
+    runs-on: self-hosted
+    env:
+      ImageOS: ubuntu22
```

Comment:
what is this variable used for?

Reply:
In self-hosted runner we need to specify the ubuntu image to use (I'm not sure why to be honest, but it doesn't work without)